projects/packages/sync/src/class-package-version.php

```diff
@@ -12,7 +12,7 @@
  */
 class Package_Version {
 
-	const PACKAGE_VERSION = '1.50.2-alpha';
+	const PACKAGE_VERSION = '1.51.1-alpha';
```

Comment:
Looks like you'll need to update this PR with latest trunk once more, and run the script to update the versions.